docs/install.rst

```diff
@@ -70,122 +97,150 @@ by executing the following command in your conda env:
-download and run the rabbitmq instance using the following command:
+        Installing rabbitmq as a system process (sudo privileges required)
+        ------------------------------------------------------------------
 
-.. code-block:: bash
+        Ensemble Toolkit relies on RabbitMQ for message transfers. Installation
+        instructions can be found at <https://www.rabbitmq.com/download.html>. At
+        the end of the installation run ```rabbitmq-server``` to start the server.
+        RabbitMQ needs to be installed on the same machine as EnTK is installed.
 
-        docker run -d --name <name of instance> -P rabbitmq:3
+        In some cases, you might have to explicitly start the rabbitmq-server after
+        installation. You can check if the rabbitmq-server process is alive. If not,
+        please run the following:
 
+        .. code-block:: bash
 
-The '-P' argument auto maps new ports from localhost to the ports expected by
-rabbitmq. This is useful if you want to have multiple EnTK scripts running as
-you would require multiple rabbitmq instances.
+                rabbitmq-server -detached
 
-You can see the mapping of the ports running ```docker ps```.
 
-.. code-block:: bash
+        Installing rabbitmq using docker
+        --------------------------------
 
-        vivek@two:~$ docker run -d --name rabbit-1 -P rabbitmq:3
-        fb8ee8bfd822656a6338b7c19fa6a9641944f8bf5de5c1414fb78d049fdffc42
-        vivek@two:~$ docker ps
-        CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS              PORTS                                                                                                 NAMES
-        fb8ee8bfd822        rabbitmq:3          "docker-entrypoint..."   9 seconds ago       Up 7 seconds        0.0.0.0:32777->4369/tcp, 0.0.0.0:32776->5671/tcp, 0.0.0.0:32775->5672/tcp, 0.0.0.0:32774->25672/tcp   rabbit-1
+        If installing rabbitmq directly seems to be cumbersome, you can also install a
+        docker instance of rabbitmq. Assuming you have docker installed, you can
+        download and run the rabbitmq instance using the following command:
 
+        .. code-block:: bash
 
-Interactions between RabbitMQ and EnTK are done through port 5672 by default.
-For the above docker instance, we need to use port 32775. In your EnTK scripts,
-while creating the AppManager, you need to specify port=32775.
+                docker run -d --name <name of instance> -P rabbitmq:3
 
-.. note::
-   If you are using Docker to install both EnTK and RabbitMQ, they should run
-   as two different containers. You can set the RMQ_PORT in the EnTK container
-   accordingly.
 
-Installation Video
-==================
+        The '-P' argument auto maps new ports from localhost to the ports expected by
+        rabbitmq. This is useful if you want to have multiple EnTK scripts running as
+        you would require multiple rabbitmq instances.
```

Comment:
I would eliminate `This is useful if you want to have multiple EnTK scripts running as you would require multiple rabbitmq instances` as I think this is not true anymore. Multiple EnTK should be able to share the same rabbitmq server.